internals/webpack/webpack.base.babel.js

```diff
@@ -124,6 +124,7 @@ module.exports = options => ({
       NODE_ENV: JSON.stringify(process.env.NODE_ENV),
       GIT_BRANCH: JSON.stringify(process.env.GIT_BRANCH) || 'dummy',
       BUILD_ENV: JSON.stringify(process.env.BUILD_ENV) || 'development',
+      ENABLE_SERVICEWORKER: JSON.stringify(process.env.ENABLE_SERVICEWORKER) || '1',
```

Comment:
Can we use a boolean value here?

Reply:
Probably not, environment variables are always a string.

Reply:
You could do something like

```
      ENABLE_SERVICEWORKER: Boolean(JSON.stringify(process.env.ENABLE_SERVICEWORKER)),
```

or

```
      ENABLE_SERVICEWORKER: JSON.stringify(process.env.ENABLE_SERVICEWORKER) === '1',
```

Reply:
That could work.

Reply:
Hmm thought of this a bit more, I think I am fan of keeping this a string as the result of this plugin is exposed as enviroment variable in Javascript, through `process.env`. I would expect this to be a string, not a boolean.